extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -517,43 +530,47 @@ angular.module('oppia').component('svgFilenameEditor', {
           ctrl.data.savedSvgFileName.length > 0);
       };
 
+      var getStrokeWidth = function() {
+        var size = ctrl.fabricjsOptions.size;
+        // Removes the word "px" from the end of the string.
+        return parseInt(size.substring(0, size.length - 2));
```

Comment:
Looks like you can just do this:
```suggestion
        return parseInt(size);
```
(see SO [answer](https://stackoverflow.com/a/4860249))

Reply:
Done.